gameros/packages.x86_64

```diff
@@ -28,3 +28,4 @@ usbutils
 # wpa_supplicant
 wget
 zsh
+networkmanager
```

Comment:
I'd suggest adding some wireless drivers like in #28 and removing the commented out entries. Although these should also need to be added to the base install then.

Reply:
Done